frontend/package.json

```diff
@@ -3,17 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bulma": "^0.7.2",
+    "bulma-extensions": "^4.0.1",
     "classnames": "^2.2.6",
+    "mediaquery": "0.0.4",
+    "node-sass-chokidar": "^1.3.4",
+    "npm-run-all": "^4.1.5",
     "prop-types": "^15.6.2",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
-    "react-scripts": "2.1.3"
+    "react-scripts": "2.1.3",
+    "redux": "^4.0.1",
+    "styled-components": "^4.1.3"
   },
   "proxy": "http://localhost:4000",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
+    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
+    "start-js": "react-scripts start",
+    "start": "npm-run-all -p watch-css start-js",
+    "build-js": "react-scripts build",
+    "build": "npm-run-all build-css build-js",
```

Comment:
Aaaaaaaaaall this should be unnecessary, because it is handled by create-react-app automatically (if you install `node-sass` instead of `node-sass-chokidar`)

Reply:
I'll try it, thanks for the feedback :100: